TrackletAlgorithm/InputRouter.h

```diff
@@ -250,7 +255,7 @@ void InputRouter( const BXType bx
 	  if( hVldBt == 0 ){ 
 	  	#ifndef __SYNTHESIS__
```

Comment:
same comment re indents. I realize this is not in code that you changed (?) but let's take care of this now too.

Reply:
Sure thing, should I do this for the whole file? There are several #pragma lines that aren't in the left margin.